app/controllers/works_controller.rb

```diff
@@ -161,6 +161,8 @@ def drafts
       redirect_to logged_in? ? user_path(current_user) : new_user_session_path
       return
     end
+    
+    @page_subtitle = ts("%{username} - Drafts", username: @user.login)
```

Comment:
Please follow the suggestion by rubocop regarding i18n. The wiki page on [i18n](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards) should help explain how it works, but the gist is that this should be a `t()` call with a locale key and the text itself should be in the locale file (`/config/locales/controllers/en.yml`).

Reply:
Hi, updated based on the i18n format. 
I think it's worth noting that all the errors and other page subtitles on the work_controller file do not follow this format.

Reply:
Indeed, there is a lot of old code that doesn't follow the latest style guidelines and i18n guidelines. That's why we have tools like rubocop to automatically notify us when new or modified code needs to be adjusted to follow the new guidelines, so we can convert the code over as we go.